src/encoded/types/shared_calculated_properties.py

```diff
@@ -284,7 +284,17 @@ class CalculatedVisualize:
     def visualize(self, request, hub, accession, assembly, status, files):
         hub_url = urljoin(request.resource_url(request.root), hub)
         viz = {}
-        for assembly_name in assembly:
+        vis_assembly = set()
+        viewable_file_formats = ['bigWig', 'bigBed']
+        viewable_file_status = ['released','in progress']
+
```

Comment:
This is a python thing, but in general don't add empty space in python functions.   It's a visual thing.  Two spaces between functions, no spaces in functions.   If it's a class, then one space in between functions.

Reply:
https://www.python.org/dev/peps/pep-0008/#blank-lines

Reply:
Thanks! @forresttanaka  It's good to have references.  I've never actually read this one.

@weiweizhong2018 Don't try to follow PEP8 word for word.  It is just a set of good team develop practices.  The point is to provide general guidelines to improve readability through code consistency.  Of course in order to accomplish that goal for a wide variety of teams PEP8 will need to be a little ambiguous.  Every instance of a team will need to use PEP8 as a guide to setup their own specific standards.

More details below, if needed.  Don't feel obligated to read it.  :)

For example,
```
Use blank lines in functions, sparingly, to indicate logical sections.
```
PEP8 needs to be vague to allow for the needs of a wide variety of teams.  But we are a specific team with individual team needs.  Senior wranglers, associate wranglers, junior and senior developers will all have a different take on words like 'sparingly' and 'logical sections'.  Given our team, the easiest application of this rule is to say, 'No spaces in functions'.  It's a simple, unambiguous rule that leads to a less chaotic code base.

In PRs, when I see spaces before **return statements**, **for loops**, **if blocks** or after function declarations, I wonder if it was on purpose, by accident, or due to a misunderstanding in how python uses indentation to structure code.  A lot of people learned to program in Java or C++ where spaces were needed for code readability.  Python is a little different and doesn't usually require spaces around common elements because indents are required instead.     

Please put this on the lower end of your coding priorities however.  The most important thing to remember is that someone will be reading and updating your code in the future.  And not to make unnecessary changes.  In the long run this will make our lives easier.